### ISSUE TITLE
Redundant provider question for python-jaxlib

### ISSUE THREAD

**Issue by xeruf** [Jguer/yay#2214]
### Affected Version

yay v12.0.5 - libalpm v13.0.2

### Describe the bug

I launch yay, and get asked for a totally superfluous choice before making the real choice.

### Expected behavior

Only second question.

### Output
```sh
❯ yay
[sudo] password for janek: 
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 multilib is up to date
:: Searching AUR for updates...
:: There are 2 providers available for python-jaxlib:
:: Repository AUR
    1) python-jaxlib 2) python-jaxlib 

Enter a number (default=1): 
==> 
:: There are 2 providers available for python-jaxlib>=0.4.12:
:: Repository AUR
    1) python-jaxlib 2) python-jaxlib-cuda 

Enter a number (default=1): 
==> 
```

**Comment by Jguer** [comment]
Hey, can you try this in `yay-git` and if it doesn't work share the output of

`pacman -Qi python-jaxlib`

**Comment by xeruf** [comment]
problem is, I updated it already now, just wanted to report it :/

**Comment by Jguer** [comment]
:) This should be fixed with #2216
